webapp/src/widgets/SortingCustomIcon/SortingCustomIcon.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Image from 'react-bootstrap/Image'
+import Icons from '../../assets/icons'
+
+const SortingCustomIcon = (order, row) => {
+	return (
+		<span className='order-arrow d-flex flex-column'>
+			{(order === 'asc' || !order) && (
+				<span className='arrow-up'>
+					<Image
+						src={row.userColumn ? Icons.caretDarkIcon : Icons.caretIcon}
+						alt='caret-icon'
+						className='opacity-50 upArrow'
+						width='10'
+					/>
+				</span>
+			)}
+			{(order === 'desc' || !order) && (
+				<span className='arrow-down'>
```

Comment:
i think only 2 classes are different
we can use npm classNames to add conditional classes rather than separate `<span>`